### ISSUE TITLE
Messages: UI/UX issues while chatting/messaging on iPhone

### ISSUE THREAD

**Issue by rabbi08** [FightPandemics/FightPandemics#1949]
1. On iPhone, Open FP application and login as valid user -> navigate to Inbox
2. Select any user to chat

Issue: You will observe that when user clicks in the textbox to type the the inbox zooms out and user need to zoom in again to click on 'Send' icon, its all the time zoom in-out. Not smooth user experience in the inbox. 
also, when user clicks on 'message' icon some user's avatar's turn grey/loading/refresh.
Refer below GIFs.

![ezgif com-video-to-gif 35](https://user-images.githubusercontent.com/67168400/100394269-7c691600-300a-11eb-9124-c8b2d8cb27fe.gif)
![ezgif com-video-to-gif 36](https://user-images.githubusercontent.com/67168400/100394277-7ffc9d00-300a-11eb-9777-b092c30a50ef.gif)

**Comment by emtay** [comment]
I saw that the inbox also 'floated' and I had to scroll to the right to see the dates (didn't fit in a single view). I have iphone 6s

**Comment by rabbi08** [comment]
@Zaydme 
I have observed that in STG today I am not seeing this UI/UX issue on iphone but it exist in PROD. Please see GIF below from both ENV.
We need to investigate how this is fixed.
![ezgif com-video-to-gif 38](https://user-images.githubusercontent.com/67168400/100488931-303bd580-30df-11eb-9db7-b67f274e9133.gif)

**Comment by Zaydme** [comment]
@rabbi08 
Not sure what PR fixed that

**Comment by emtay** [comment]
I noticed that it was fixed in staging too! I assumed Zayd worked his magic! :)

**Comment by Zaydme** [comment]
Okay closing this, but we need to reopen if it happened again.

**Comment by emtay** [comment]
Good idea. I'll keep testing on iOS

**Comment by rabbi08** [comment]
@Zaydme 
This issue(UI/UX experience on iPhone) exist in Production but NOT in STG, hence reopening for further investigation.

Please see below GIF.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/67168400/102383982-2ac50300-3f9a-11eb-908a-44accbfc89a4.gif)

**Comment by Zaydme** [comment]
Yeah now I think this is caused by the floating accessibility icon since it only exists in production
